addons/discuss-button/addon.json

```
@@ -96,6 +105,12 @@
       "type": "boolean",
       "default": false
     },
+    {
+      "name": "Expand contents to the edges",
```

I have some name ideas:
```suggestion
      "name": "Span the entire page's width",
```
```suggestion
      "name": "Extend to page edges",
```

---

I think the second suggestion here is the best so far.